client/src/frontend/components/atomic/EntityHeadContent/index.js

```diff
@@ -57,6 +66,7 @@ function EntityHeadContent({ entity, parentEntity = { attributes: {} } }) {
 
 EntityHeadContent.propTypes = {
   entity: PropTypes.object.isRequired,
+  type: PropTypes.string,
```

Comment:
We could use `entity.type` but we'd have to format the strings, so I decided to hard code the prop. Could be a bad decision.